### PR TITLE
Prepare release 1.0.0

### PR DESCRIPTION
**Pull request by lemunozm**
Prepare release 0.2.0. Once it has been merged, I will upload it to cargo repositories.

### CODE REVIEW

**Comment by lemunozm** [comment]
Hi, @sigmaSd, could you confirm the changelog file?

Also, if you want, you can suggest the `authors` line in `cargo.toml` with your name and email :)

I left the `send_files` feature for the next version when message-io works fine in all border cases

File: CHANGELOG.md [hunk 1]
```diff
@@ -0,0 +1,12 @@
+# Changelog
+
+## Release 0.2.0
+- Fixed network issue that avoid to connect termchat through the LAN.
```
Comment (sigmaSd) [inline CHANGELOG.md]: Fixed a network issue that made peers not discoverable through LAN.

**Comment by sigmaSd** [comment]
Looks fine to me^^

**Comment by sigmaSd** [comment]
Also one suggestion, you might want to make this release 1.0.0, 
Termchat already does what it advertises, from now on we're just adding new features.

File: CHANGELOG.md (hunk 1 above)
Comment (lemunozm) [reply CHANGELOG.md]: Updated :)

**Comment by lemunozm** [comment]
> Also one suggestion, you might want to make this release 1.0.0,
> Termchat already does what it advertises, from now on we're just adding new features.

I agree. Updated :). 

Will you want to modify the author's line before I merge it? (You can create a *suggestion* for this change)

**Comment by sigmaSd** [comment]
Thanks but I prefer not to

But I really appreciate the suggestion!

**Comment by lemunozm** [comment]
Ready in cargo repositories :)

**Pull request merged**